src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/app_start/Router.kt

```diff
@@ -48,16 +53,24 @@ class Router(freeMarkerConfig: Configuration)
         urls.addAll(routeConfig.endpoints.map { mapEndpoint(it, urlBase) })
     }
 
-    private fun mapLoginCallback()
+    private fun mapLoginCallback(config: Config)
     {
-        val config = WebSecurityConfigFactory(MontaguIndirectClient(), setOf())
-                .build()
         val loginCallback = CallbackRoute(config)
         val url = "login"
         Spark.get(url, loginCallback)
         Spark.get("$url/", loginCallback)
     }
 
+    private fun mapLogoutCallback(config: Config)
+    {
+        val logoutCallback = LogoutRoute(config)
+        logoutCallback.destroySession = true
+        logoutCallback.defaultUrl = "/"
```

Comment:
This will throw us out to the auth provider after logging out of here, but we will still be logged in there... Is this the right behaviour?

Reply:
Hmm, so a user might log out, be taken back to Montagu where they're still logged in, and then revisit OrderlyWeb and find themselves logged right back in again. Does feel like that would be a confusing user experience.

I guess the weirdness derives from the fact that we have no login screen, we just automatically direct users to the external provider to log in. So there's no logged out view of OrderlyWeb, no way for a user to visually confirm that they're logged out. Maybe we should have a landing page after all, with a prompt to click through to login? 

The Montagu case is a particularly weird one because we're disguising the fact they are separate apps so the user will expect that they are logging out of Montagu altogether. Maybe we go ahead and log them out of Montagu tbh? 

Basically I don't know what the right behaviour is :sob: